crates/bw/src/main.rs

```diff
@@ -178,32 +177,6 @@ async fn process_commands() -> Result<()> {
             }
             return Ok(());
         }
-        Commands::Register {
```

Comment:
Clippy is complaining about the single branch match, you can do two things here:
- Keep the branch, but don't do anything:
```rust
Commands::Register { .. } => unimplemented!(),
```
- Silence the warning on top of the match:
```rust
#[allow(clippy::single_match)]
match command.clone() {
```